approval_polls/views.py

```diff
@@ -36,6 +44,13 @@ def myPolls(request):
 
 
 @login_required
+def taggedPolls(request, tag):
```

Comment:
Does this need the `@login_required` ? I think it should be able to filter publicly visible polls even while the user is not logged in.